### ISSUE TITLE
The In-Hand Crusade of Anno MMXXIII

### ISSUE THREAD

**Issue by BomberBro** [d3athrow/vgstation13#34717]
#### Description of issue
We have a lot of sprites that don't have in-hand icons. I'm of the opinion that every single item should have some sort of in-hand, both for gameplay purposes (Sometimes very, very strong weapons lack an inhand so if you aren't mashing inspect on every person you can get iced instantly without any way to tell, an example being the chem sprayer) and just general ease of knowing what's going on in the game. It's also good for roleplay for some items but that's really up to the RPheads to decide. 

I have previously done a PR like this 3 years ago (fuck) in #26437 but that doesn't even begin to scratch the surface of what's missing. If you know an item that doesn't have an in-hand, and it's something that you can reasonably get (so nothing that you'd only see once in a blue moon anyway ie spessmart exclusive clothing or most adminbus items) then let me know and I'll make an in-hand for it, simple as that. I don't even care if it's something as simple as the radiation patch. If there's a desire to see an in-hand for it, let me know and I'll make it. 

**I will also make in-hands for items the current in-hand does not properly reflect (bluespace beakers, most command jumpsuits) if wanted, or even rare items that are sufficiently important.** This PR is mainly to gauge interest of what items to look at first and what are most important to people, in the end I'd ideally like every item to have one but realistically what people want should be done first in case that doesn't happen. 

[sprites]

**Comment by SonixApache** [comment]
there's a unit test for this

**Comment by djkramer123** [comment]
Not a terribly high priority item, but the alden saraposa scanner still uses the lamp in hand I'm pretty sure

**Comment by gurfan** [comment]
The plasma cutter uses the revolver inhand, and I don't like having a panic attack every time a miner looks at me.

**Comment by Eneocho** [comment]
The fruits/veggies in general, but mainly the sunflower/nettle and the mutations could use an inhand, cause a maxed nettle or sunflower can pretty much one or twohit someone, so being able to see that would be nice.
Also a head icon for wearing the nova as a hairpin would be nice, you can already do that but there's no icon for it.

**Comment by gurfan** [comment]
I wish most of the bar drinks had a custom inhand or at least used the base drinking glass inhand. Almost all of them are invisible.

**Comment by RealestEstate** [comment]
@SonixApache what's a unit test?

**Comment by Hinaichigo** [comment]
Medical messenger bag inhand is a grey backpack.

**Comment by pedr69** [comment]
an easy "fix" would be making the parent /obj/item have a shitty placeholder sprite, something like `DONGS` perhaps
![image](https://github.com/vgstation-coders/vgstation13/assets/18052467/66b209e5-3c03-4f35-b77a-1300897455d2)

fix is in quotes because this won't work on items that have their item state but not the inhand icons overridden for some retarded reason(like the chemsprayer)

**Comment by Hinaichigo** [comment]
icon_state is the icon for the item on the ground or in the inventory, it's the icon_state corresponding to  the dmi specified in icon

item_state is the inhand sprite that gets overlayed onto the sprite of the person holding it. it refers to to an icon_state in a dmi with a variable file path that's constructed dynamically depending on which creature is wearing/holding it, in which hand, if they're fat, etc. typically something like
icons/mob/in-hand/left/

**Comment by SECBATON-GRIFFON** [comment]
why is there two separate folders for left and right hands with dupe paths
i feel like a lot of the inhands could just be mirrored save for some exceptions and the left-rights can be kept for items that need it, or some dirs could have them as underlays instead. that or scaled down sprites of the items themsevles with rotation, etc.

**Comment by Eneocho** [comment]
not really, mirroring the sprite isn't enough, you have to move it a few pixels as well
that being said, you could probably just automate it such that right hand sprite gets mirrored then X-shifted by some pixels
